lib/core/macros.nim

```diff
@@ -1545,7 +1545,12 @@ proc customPragmaNode(n: NimNode): NimNode =
               let varNode = identDefs[i]
               # if it is and empty branch, skip
               if varNode[0].kind == nnkNilLit: continue
-              if varNode[1].kind == nnkIdentDefs:
+              # Skiping, because required ident node is only available in node of kind nnkOfBranch
```

Comment:
```suggestion
              # Skipping, because required ident node is only available in node of kind nnkOfBranch
```

Reply:
I force pushed my changes since the logic I implemented here wasn't right.